mitmproxy/addons/modifyheaders.py

```diff
@@ -81,7 +82,7 @@ def parse_modify_spec(option, subject_is_regex: bool, replacement_is_path: bool)
     if replacement_is_path:
         path = Path(replacement)
         try:
-            replacement = path.expanduser().resolve(strict=True)
+            replacement = str(path.expanduser().resolve(strict=True))
```

Comment:
Hm, Windows and OSX tests for `MapLocal` fail due to the `resolve()` and the obviously non-existing `/tmp` path that the `test_file_candidates` test uses. We do this similarly for the other "replacement" addons when an option with `@path` is provided (see https://github.com/mitmproxy/mitmproxy/blob/master/mitmproxy/addons/modifyheaders.py#L26), but `parse_modify_spec()` is never tested with an `@path` option. Therefore it's not an issue there.

Should I move the `resolve()` check outside of `parse_modify_spec()` or use the `tmpdir` of pytest?

Reply:
I would probably try `tmpdir`, but either option works for me :)